app/Models/Feed.php

```diff
@@ -695,7 +695,11 @@ public function pubSubHubbubPrepare() {
 
 	//Parameter true to subscribe, false to unsubscribe.
 	public function pubSubHubbubSubscribe($state) {
```

Comment:
not a big deal and not directly related to this PR, but wouldn't it better to call this method `setPubSubHubbubSubscriptionState`? or `setWebsubSubscriptionState`?

Reply:
Yes, a bit of renaming at a few places could be good. I will try to find time to make another PR for better handling the case when a user deletes a feed, in a multi-user context